project/NativeImage.scala

```diff
@@ -240,8 +242,8 @@ object NativeImage {
           Seq("-jar", pathToJAR.toString)
       }
 
-      val targetDir = (Compile / target).value
-      val argFile   = targetDir.toPath.resolve(NATIVE_IMAGE_ARG_FILE)
+      val td      = (Compile / target).value
```

Comment:
Please don't make such name changes. This is hard to read.

Reply:
bbbbf25b98